webapp-django/crashstats/crashstats/tests/test_models.py

```diff
@@ -400,14 +400,11 @@ def mocked_get(**params):
                 "Theme": "classic/1.0",
                 "Version": "5.0a1",
                 "Vendor": "Mozilla",
-                # This is not allowlisted
-                "URL": "https://example.com/",
             }
 
         model.implementation().get.side_effect = mocked_get
         r = api.get(crash_id="some-crash-id")
         assert r["Vendor"] == "Mozilla"
-        assert "URL" in r  # no filtering at this level
```

Comment:
This is different now. If you don't explicitly set an `api_user` that has protected-data permissions, then you now get a filtered public-only set of data. public-by-default.